npm/private/npm_package.bzl

```diff
@@ -440,8 +442,14 @@ def npm_package(
             srcs = srcs,
             include_sources = include_sources,
             include_transitive_sources = include_transitive_sources,
-            include_declarations = include_declarations,
-            include_transitive_declarations = include_transitive_declarations,
+            include_declarations = select({
+                "@aspect_rules_js//npm:exclude_declarations_from_npm_packages_flag": False,
+                "//conditions:default": include_declarations,
+            }),
+            include_transitive_declarations = select({
```

Comment:
Would only setting this one to `False` ever be an improvement on its own? 🤔

Reply:
I don't think so, because producing the direct .d.ts inputs will run the typechecker and it will use the transitive .d.ts as inputs, so they'll be requested from Bazel as well.

Reply:
Yeah, that's the standard case, I'm trying to think if there are any others 🤔 🤷